roles/bahmni-emr/templates/mysql56.repo.j2

```diff
@@ -0,0 +1,6 @@
+[mysql56-community]
+name=MySQL 5.6 Community Server
+baseurl=http://repo.mysql.com/yum/mysql-5.6-community/el/7/x86_64/
+enabled=1
+gpgcheck=0
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-mysql
```

Comment:
this is just for example sake? or are you actually placing the file there?

Reply:
@angshu its not an example..earlier we used to have only one mysql.repo.j2 specific to mysql 5.7. But here we have separate files each for mysql 5.6 and mysql 5.7.